AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -157,7 +165,22 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
             Tests.redo_material_asset_color_change,
             material_editor_utils.get_property(document_id, base_color_property_name) == expected_color)
 
-        # 10. Look for errors and asserts.
+        # 10. Change the lighting background displayed behind the material model in the viewport.
+        lighting_background_asset_path = os.path.join(
+            "@gemroot:MaterialEditor@", "Assets", "MaterialEditor", "LightingPresets",
+            "neutral_urban.lightingpreset.azasset")
+        lighting_background_asset = atom_tools_utils.load_lighting_preset(lighting_background_asset_path)
+        Report.result(
+            Tests.viewport_background_changed, lighting_background_asset is True)
+
+        # 11. Change the material model displayed in the viewport.
+        model_asset_path = os.path.join(
+            "@gemroot:MaterialEditor@", "Assets", "MaterialEditor", "ViewPortModels", "BeveledCone.modelpreset.azasset")
+        model_asset = atom_tools_utils.load_model_preset(model_asset_path)
+        Report.result(
+            Tests.viewport_model_changed, model_asset is True)
```

Comment:
load_model_preset returns true if the path either caches or deserializes AZ::Render::ModelPreset struct. This test mostly confirms that "BeveledCone.modelpreset.azasset" is AZ::Render::ModelPreset and we can then assume the model preset has been updated.
I'm not sure how we would determine the asset currently loaded from python but I feel like there are missing ebus calls to determine the assets currently active in these values as something other than C++ struct addresses. From a testing perspective we should be able to set and get then compare the return of the get against what we set to ensure that the set worked rather than taking the set's return of True as evidence of successfully updated.
The distinction is probably too minor to require a change.

Reply:
i will add an asset check but this check is valid as is. i tested `load_model_preset()` same as i did for `load_lighting_preset()` and it does indeed return False if it doesn't load the preset, True if it does

but an asset check as part of the test is good too, so ill add it